package.json

```diff
@@ -25,7 +25,8 @@
     "heroku-cli-util": "5.10.10",
     "ignore": "3.1.2",
     "node-uuid": "1.4.7",
-    "request": "2.72.0"
+    "request": "2.72.0",
```

Comment:
Thank you, you're right. I'll remove it.
I'd rather keep this PR focused on the tests though.